MediaWatermark/Classes/Extensions/Assets.swift

```diff
@@ -18,25 +18,34 @@ extension AVAsset {
         return isContentPortrait ? CGSize(width: contentNaturalSize.height, height: contentNaturalSize.width) : contentNaturalSize
     }
     
-    var contentOrientation: UIInterfaceOrientation {
-        guard let transform = tracks(withMediaType: AVMediaType.video).first?.preferredTransform else {
-            return .portrait
+    var contentOrientation: UIImageOrientation {
+        var assetOrientation = UIImageOrientation.up
+        let transform = tracks(withMediaType: AVMediaType.video)[0].preferredTransform
+        
+        if(transform.a == 0 && transform.b == 1.0 && transform.c == -1.0 && transform.d == 0)
+        {
```

Comment:
Please format this code block. We should have opening brackets on the same line as if statement

Reply:
Or you can integrate Swift Lint library to avoid similar issues with code formatting in future

Reply:
https://github.com/realm/SwiftLint